R/pkg/R/functions.R

```diff
@@ -2875,9 +2875,9 @@ setMethod("ifelse",
 #' @details
 #' \code{cume_dist}: Returns the cumulative distribution of values within a window partition,
 #' i.e. the fraction of rows that are below the current row:
-#' number of values before (and including) x / total number of rows in the partition.
+#' (number of values before and including x) / (total number of rows in the partition)
 #' This is equivalent to the \code{CUME_DIST} function in SQL.
```

Comment:
This is not a formula, right? Thought this is pretty clear. Not sure what is the ask. I can just add a period after the `partition)`.